cql3/restrictions/statement_restrictions.cc

```diff
@@ -1178,6 +1178,45 @@ bool is_satisfied_by(
         }, restr);
 }
 
+bytes_opt get_bound(const expression& restr, const query_options& options, statements::bound bnd) {
+    return std::visit(overloaded_functor{
+            [&] (const conjunction& conj) {
+                if (!conj.children.empty()) {
+                    auto invoke_get_bound = [&] (const ::shared_ptr<expression>& p) {
+                        return get_bound(*p, options, bnd);
+                    };
+                    // For now, assume conjunction elements all have the same LHS.
```

Comment:
How can we assume it?

Reply:
Because of how existing restrictions are currently created: single_column_restrictions maintains a map from each restricted column to its conjunction.  See [single_column_restrictions.hh:150](https://github.com/scylladb/scylla/blob/master/cql3/restrictions/single_column_restrictions.hh#L150).

Reply:
Can we reword to "single_column_restrictions::add_restriction() guarantees that"? It will help future readers (including me)

Reply:
Reworded.